.github/workflows/sonar.yml

```
@@ -0,0 +1,46 @@
+      - name: SonarCloud Scan
+        uses: SonarSource/sonarqube-scan-action@v5
+        env:
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+        with:
+          args: >
+            -Dsonar.host.url=https://sonarcloud.io
+            -Dsonar.projectKey=Learnathon-By-Geeky-Solutions_ode-to-code
+            -Dsonar.organization=learnathon-by-geeky-solutions
+            -Dsonar.java.version=17
+            -Dsonar.verbose=true
+            -Dsonar.dart.coverage.reportPath=coverage/lcov.info
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)